bundles/org.openhab.core.automation/src/main/java/org/openhab/core/automation/util/ActionInputsHelper.java

```diff
@@ -148,10 +151,12 @@ public ConfigDescriptionParameter mapActionInputToConfigDescriptionParameter(Inp
                     break;
                 case "java.time.LocalTime":
                     context = "time";
+                    step = BigDecimal.ONE;
```

Comment:
what does one in this context mean? one second?

Reply:
Exactly.
The step size for datetime and time contexts defaults to 60 seconds, setting it to 1 second enables them to display seconds as well.